apps/www/components/cookie-banner.tsx

```
@@ -0,0 +1,108 @@
+"use client";
+
+import {CookieBanner, ConsentManagerDialog} from "@c15t/react";
+
+const CookieBannerComponent = () => {
+  const secondaryButton =
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend secondaryButton with an underscore.


(lint/correctness/noUnusedVariables)